examples/cloud-router-2-metal-nimf-redundant-connection/terraform.tfvars.example

```diff
@@ -0,0 +1,27 @@
+equinix_client_id     = "<MyEquinixClientId>"
+equinix_client_secret = "<MyEquinixSecret>"
+metal_auth_token      = "<Metal_Auth_Token>"
+
+metal_connection_metro      = "sv"
+metal_project_id            = "<Metal_Project_ID>"
+metal_connection_name       = "Metal-NIMF-connection"
+metal_connection_redundancy = "redundant"
+metal_connection_speed      = "50Mbps"
+metal_connection_type       = "shared_port_vlan"
+metal_contact_email         = "tfacc@example.com"
+
+connection_name                  = "FCR_2_Metal_DIGP"
+connection_type                  = "IP_VC"
+notifications_type               = "ALL"
+notifications_emails             = ["example@equinix.com","test1@equinix.com"]
+purchase_order_number            = "1-323292"
+bandwidth                        = "50"
+project_id                       = "<Fabric_Project_ID>"
+zside_ap_type                    = "METAL_NETWORK"
+primary_direct_rp_name           = "pri_direct_rp"
+secondary_direct_rp_name         = "sec_direct_rp"
+primary_direct_equinix_ipv4_ip   = "190.1.1.25/30"
+secondary_direct_equinix_ipv4_ip = "190.1.1.9/30"
```

Comment:
I would reorder the example variables to match the ordering present in variables.tf files.

Reply:
Addressed with [commit](74747cd2cf07ab77431eb1de6c1f69e1749e578a)